charts/incubator/pbs/Chart.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: v2
+appVersion: "2.2-7"
+dependencies:
+  - name: common
+    repository: https://library-charts.truecharts.org
+    version: 13.2.1
+description: Container for proxmox backup server
+home: https://truecharts.org/charts/incubator/proxmox-backup-server
+icon: https://truecharts.org/img/hotlink-ok/chart-icons/pbs
+keywords:
+  - proxmox-backup-server
+  - pbs
+kubeVersion: ">=1.16.0-0"
+maintainers:
+  - email: info@truecharts.org
+    name: TrueCharts
+    url: https://truecharts.org
+name: pbs
```

```suggestion
name: proxmox-backup-server
```
Please rename the directory too